spacewalk/setup/spacewalk-setup.spec

```diff
@@ -196,10 +196,10 @@ mkdir -p $RPM_BUILD_ROOT%{_mandir}/man8
 /usr/bin/pod2man --section=1 $RPM_BUILD_ROOT/%{_bindir}/spacewalk-setup-ipa-authentication| gzip > $RPM_BUILD_ROOT%{_mandir}/man1/spacewalk-setup-ipa-authentication.1.gz
 
 # Standalone Salt formulas configuration
-install -Dd -m 0750 %{buildroot}%{_prefix}/share/salt-formulas
-install -Dd -m 0750 %{buildroot}%{_prefix}/share/salt-formulas/states
-install -Dd -m 0750 %{buildroot}%{_prefix}/share/salt-formulas/metadata
-%
+install -Dd -m 0755 %{buildroot}%{_prefix}/share/salt-formulas
+install -Dd -m 0755 %{buildroot}%{_prefix}/share/salt-formulas/states
+install -Dd -m 0755 %{buildroot}%{_prefix}/share/salt-formulas/metadata
```

Comment:
This needs to be changed in the filelist as well.

Reply:
Thanks, changed it in the filelist and created separate commits for fixing both errors (syntax error and directories).